### PR TITLE
Separated public_key and submission_url, closes #146

### LINKED ISSUE #146: Encryption and Submission URL are two separate unrelated things
Separate them in the text.

### PR DESCRIPTION
**Pull request by MartijnR**

### CODE REVIEW

File: _sections/home-english.md [hunk 1]
```diff
@@ -748,27 +748,32 @@ An example **settings** worksheet is below:
...
+* **submission_url**: This url can be used to override the default server where finalized records are submitted to.
 * **default_language**: In localized forms, this sets which language should be used as the default.
 * **version**: String of up to 10 numbers that describes this revision. Revised form definitions must have alphabetically greater versions than previous ones. A common convention is to use strings of the form 'yyyymmddrr'. For example, 2017021501 is the 1st revision from Feb 15th, 2017.
 
 ### Encrypted forms
 
-Encrypted forms provide a mechanism to keep your data private using http for communication. Form submissions sent to the Aggregate server are encrypted and completely inaccessible to anyone not possessing the private key.
+Encryption-enabled forms provide a mechanism to keep finalized records private at all times. This includes the time records are stored on the device and server as well as during transport, even when http is used for communication. Encrypted records including their uploaded files, such as photos, are completely inaccessible to anyone not possessing the private key.
```
Comment (lognaturel) [inline _sections/home-english.md]: I'm wondering whether this is a little bit ambiguous? "finalized records" is specified in the first sentence but then the second sentence could be taken to imply records are encrypted as they're being filled out which is not the case across clients (could get very slow). To be even more explicit, maybe something like "This includes the time after finalization that records are..." Or alternately "Note that un-finalized records are not (may not be?) stored encrypted."

File: _sections/home-english.md [hunk 2]
```diff
@@ -748,27 +748,32 @@ An example **settings** worksheet is below:
...
+| form_id | public_key     | 
+| ------- | -------------- | 
+| mysurvey   | IIBIjANBgklawWEserewrwesgdreewrwe32serfserfewrwerewtwer23sgfrqjwerk3423432...   | 
+| ======  | ============== | 
+| settings   |         |     
 
-  * **form_id** - name used to identify the form
-  * **submission_url** - is your submission url
-  * **public_key** - is the base64RsaPublicKey
+For more information on encrypted forms and how to generate the RSA keys have a look at the [ODK documentation](http://opendatakit.org/help/encrypted-forms/) and at [this this sample forms](https://docs.google.com/spreadsheets/d/1O2VW5dNxXeyr-V_GB3spS6QPX4rtqtt7ijqP_uZLU3I/edit?usp=sharing) example.
```
Comment (lognaturel) [inline _sections/home-english.md]: Could avoid the redirect here by linking directly to https://docs.opendatakit.org/encrypted-forms/

File: _sections/home-english.md (hunk 1 above)
Comment (MartijnR) [reply _sections/home-english.md]: yes, very good

File: _sections/home-english.md [hunk 3]
```diff
@@ -767,7 +767,7 @@ To encrypt XLS forms, add the **public_key** column to the **settings** workshee
 | ======  | ============== | 
 | settings   |         |     
 
-For more information on encrypted forms and how to generate the RSA keys have a look at the [ODK documentation](http://opendatakit.org/help/encrypted-forms/) and at [this this sample forms](https://docs.google.com/spreadsheets/d/1O2VW5dNxXeyr-V_GB3spS6QPX4rtqtt7ijqP_uZLU3I/edit?usp=sharing) example.
+For more information on encrypted forms and how to generate the RSA keys have a look at the [ODK documentation](https://docs.opendatakit.org/encrypted-forms/) and at [this this example form](https://docs.google.com/spreadsheets/d/1O2VW5dNxXeyr-V_GB3spS6QPX4rtqtt7ijqP_uZLU3I/edit?usp=sharing).
```
Comment (lognaturel) [inline _sections/home-english.md]: Haha, how did I not catch "this this" before (not your error)! If you're at your screen, could be a quick improvement to make there before merging.

Comment (MartijnR) [reply _sections/home-english.md]: OMG! I edited that part... :(. Will do right away! Thanks!

Comment (lognaturel) [reply _sections/home-english.md]: The plural "forms" must have been for the double "this". ;)

**Pull request merged**